kotsadm/web/src/components/UploadAirgapBundle.jsx

```diff
@@ -361,7 +361,7 @@ class UploadAirgapBundle extends React.Component {
 
   getApp = async (slug) => {
     try {
-      const res = await fetch(`${window.env.API_ENDPOINT}/apps/app/${slug}`, {
+      const res = await fetch(`${window.env.API_ENDPOINT}/app/${slug}`, {
```

Comment:
route is not updated in the api

Reply:
good catch. i tried to update this but it didnt work and then i tried to roll it back but apparently i didnt